src/main/webapp/assets/feedScript.js

```diff
@@ -77,15 +77,24 @@ document.addEventListener('DOMContentLoaded', onLoad);
 /**
  * Fires as soon as the DOM is loaded.
  */
-function onLoad() {
+async function onLoad() {
+  const collegeid = (new URLSearchParams(window.location.search)).get('collegeid');
+
+  // Don't load the map, posts if there wasn't a college ID provided.
+  if (!collegeid) {
+    document.getElementById('find-events-title').innerHTML =
+      'No college provided. Please visit the <a href="/">home page</a> to pick a college.';
```

Comment:
Since all titles are lowercase, should this be lowercase as well?

Reply:
Ok!